doc/design/intra-process.md

```diff
@@ -0,0 +1,83 @@
+# Intra-process delivery
+
+Several requirements, each with a section.
+
+## Identifying local endpoints
+Endpoints on the same process have the same content on the first 8 octets of their GUID prefix. 
```

Comment:
This is not true when -DSECURITY=ON. In that case the final GUID_t has inside a MD5 of all bytes of the original participant GUID_t.

Reply:
I have added a comment on additional considerations section